src/gsAssembler/gsVisitorMass.h

```diff
@@ -76,9 +76,19 @@ class gsVisitorMass
                          gsGeometryEvaluator<T> & geoEval,
                          gsVector<T> const      & quWeights)
     {
+        // /* // standard version
+        for (index_t k = 0; k != quWeights.rows(); ++k) // loop over quadrature nodes
```

Comment:
do we know if the change has any effect on the efficiency?

Reply:
This was unintentional, will revert it